install/kubernetes/helm/istio/values.yaml

```diff
@@ -5,6 +5,13 @@
 gateways:
   enabled: true
 
+#
+# ingress gateway agent configuration
+#
+ingressGatewayAgent:
```

Comment:
Why is this top level ? It's a feature flag for gateway.

Reply:
It has been moved into install/kubernetes/helm/subcharts/gateways/values.yaml